http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200817_add_ksa_and_gsa_to_app_table.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="gabriela" id="add_ksa_and_gsa_to_app_table">
+        <addColumn tableName="APP">
+            <column name="googleServiceAccount" type="VARCHAR(254)">
+                <constraints nullable="false"/>
```

Comment:
Just checked in dev and there are a couple of existing `APP` records. Not sure if adding a new not-null column will break w/ existing records.. maybe we should add some dummy default value just in case.

Reply:
ah good point, i'll update then merge